### ISSUE TITLE
Please fix the issues ASAP, the chat and the composer seems to be totally DUMB why?

### ISSUE THREAD

**Issue by fisforfaheem** [getcursor/cursor#2452]
cursor seems to work like copilot now, whe chat and composer feels weird and not accurate, also APPLY NOT WOKIng

**Comment by StylezDevops** [comment]
agreed. no matter what model I select now.. it is like talking to a ten year old. tasks that claude sonnet previously handled with ease (particularly when dealing with a codebase/long context chat) - now it just asks loads of dumb follow up questions - "would you like me to proceed?" - then does half the task and asks more dumb questions with "would you like to continue" - it's awful and to the point i might even just cancel the subscription. what's the point of it now? I can get this level of stupidness for free.

**Comment by MatheusVBLima** [comment]
It is also commenting import lines for no reason. Fells like every new chat/composer addition I have to fix all the lines it commented

**Comment by noahslens** [comment]
You can circumvent 99% of this with a solid .cursorrules prompt.

**Comment by OJT-dev** [comment]
Thanks for the tip. I've actually been experimenting with creating prompts
lately, using a combination of Cline, Aider and the composer. I'll see if
refining my .cursorrules prompt improves things further.

**Comment by leerob** [comment]
Thanks for opening this issue! We've moved our community support to the [Cursor Forum](https://forum.cursor.com) where you can get help from both the team and other users.

If this is a specific bug that you can still reproduce, please open a new issue with clear reproduction steps on the forum.

Thanks you!
